content/posts/2021-04-27-server-side-rendering-with-netlify/2021-04-27-server-side-rendering-with-netlify.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: 'Server-side rendering (SSR) with Netlify'
+author: henrik
+thumbnail: './thumbnail.png'
+date: '2021-04-27T11:50:12.613Z'
+---
+
+In recent years, we have seen a slide shift from optimising everything for
+static hosting towards hybrid solutions. Next.js, and also the rising star
+Svelte Kit makes it easy to render routes statically or dynamically. This is
+just an awesome tool in our toolbox and I was curious how that works under the
+hood. While digging into Svelte’s Netlify adapter, I stumbled across an awesome,
+yet hidden feature. Today, we are going to talk about redirect rules but from a
+frontend guy perspective.
```

Comment:
from a frontend perspective w/o guy maybe? I don't know 🤷‍♂️

Reply:
Changed it :)